package.json

```diff
@@ -3,8 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+
+    "@iconify-icons/el": "^1.0.0",
+    "@iconify/icons-mdi": "^1.0.151",
+    "@iconify/react": "^1.1.3",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
```

Comment:
1. All the whitespace here is not needed -- again, must have been when dealing with the merge conflicts (this is why we want to always stay even with `main` to avoid all the merge conflicts) 
2. When dealing with merge conflicts, please be aware of what the changes are and what changes need to be merged in. I am fairly certain that PR #4 got rid of the `@testing-library/*` dependencies. They're not in our main branch 
https://github.com/CSE115A/rideshare-project/blob/614b5d4ff5dbabbe52f8a4e0aa3ce3ee998ad3d5/package.json#L5-L14
Please be careful with those as they can be tricky to manage.